x/auth/mock/simulate_block.go

```diff
@@ -53,14 +53,16 @@ func GenTx(msg sdk.Msg, seq []int64, priv ...crypto.PrivKeyEd25519) auth.StdTx {
 }
 
 // simulate a block
-func SignCheckDeliver(t *testing.T, app *baseapp.BaseApp, msg sdk.Msg, seq []int64, expPass bool, priv ...crypto.PrivKeyEd25519) {
+// expPassCheck : Set to true if you expect Check() to pass i.e. the ante handler will pass
+// expPassDeliver : Set to true if you expect Deliver() to pass i.e. the ante handler and the msg handler will pass
+func SignCheckDeliver(t *testing.T, app *baseapp.BaseApp, msg sdk.Msg, seq []int64, expPassCheck bool, expPassDeliver bool, priv ...crypto.PrivKeyEd25519) {
```

Comment:
Was getting a few CI errors with how runTx() was changed to split CheckTx() and DeliverTx(). The errors were due to having expPass set to fail for a msg handler failure. However there were a few instances where the ante handler passes, and the msg handler fails, but we were expecting both to fail in the failed CI tests. To fix this I split expPass to expPassCheck and expPassDeliver. Essentially if you expect it to pass CheckTx() (mainly the ante handler and fee functionality), you set expPassCheck to true. If it would expect Deliver() to pass (ie. the msg handler) then you'd set expPassdeliver to True